src/mainboard/protectli/vault_adl_n/variants/vp32xx/gpio.c

```diff
@@ -41,8 +40,10 @@ static const struct pad_config gpio_table[] = {
 	/* GPP_B14 - SPKR */
 	PAD_CFG_NF(GPP_B14, NONE, PLTRST, NF1),
 	PAD_NC(GPP_B15, NONE),
-	PAD_NC(GPP_B16, NONE),
-	PAD_NC(GPP_B17, NONE),
+	/* GPP_B16 - I2C5_SDA */
```

Comment:
This file needs reverts. Some GPIOs were updated on the vp3200 branch. This PR should only move the file and change the include from `"gpio.h"` to `<variant/gpio.h>`

Reply:
[Fixed](https://github.com/Dasharo/coreboot/pull/576/files#diff-e11188597ca444e2e87545dbd1cfa4ed0ef080fbb2c99201ac3c87ea8b16ecea)